### ISSUE TITLE
`npm publish` succeeded but latest version is not in the registry

### ISSUE THREAD

**Issue by mikol** [npm/npm#20851]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [x] npm is producing incorrect or undesirable behavior.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?

I  have published a package [dyad-fsa-promise](https://www.npmjs.com/package/dyad-fsa-promise) twice after `npm version patch`, but the latest version has not been updated.

Here’s my last publish success:

```
> dyad-fsa-promise@0.1.2 /Users/mikol/src/com/github/mikol/dyad-fsa-promise
> npm publish

npm notice 
npm notice 📦  dyad-fsa-promise@0.1.2
npm notice === Tarball Contents === 
npm notice 1.1kB package.json                 
npm notice 168B  CHANGELOG.md                 
npm notice 105B  dyad-fsa-promise.d.ts        
npm notice 1.2kB dyad-fsa-promise.js          
npm notice 427B  README.md                    
npm notice 254B  scripts.js                   
npm notice 375B  tsconfig.json                
npm notice 2.9kB tslint.json                  
npm notice 693B  src/dyad-fsa-promise.ts      
npm notice 3.4kB test/dyad-fsa-promise.test.ts
npm notice === Tarball Details === 
npm notice name:          dyad-fsa-promise                        
npm notice version:       0.1.2                                   
npm notice package size:  3.0 kB                                  
npm notice unpacked size: 10.7 kB                                 
npm notice shasum:        0614ba4531951ab851efcfd57594b1f9658b4f01
npm notice integrity:     sha512-I6+LgIuz9UQw+[...]YIktLOfSgG/pw==
npm notice total files:   10                                      
npm notice 
+ dyad-fsa-promise@0.1.2

```

Here’s `npm view`’s output:

```
> npm view dyad-fsa-promise

dyad-fsa-promise@0.1.0 | CC-BY-SA-4.0 | deps: 2 | versions: 4
Dyad promise middleware with Flux Standard Action support
https://github.com/mikol/dyad-fsa-promise#readme

keywords: dyad, flux, fsa, middleware, promise

dist
.tarball https://registry.npmjs.org/dyad-fsa-promise/-/dyad-fsa-promise-0.1.0.tgz
.shasum: d5a2b7948a57a3fadd473751ca1708d49653c247
.integrity: sha512-1AC8xLI1sMcnTBHFQ1hNqJpyTvPBIy41Ona95Po0eB7SJ8shiO7IYcSHxZRbRiLU4N94/Ur+7LMCFS/nfiaQ5A==
.unpackedSize: 10.7 kB

dependencies:
dyad: git+https://github.com/mikol/dyad.git flux-standard-action: ^2.0.3                

maintainers:
- mikol <mikol@thinbox.org>

dist-tags:
latest: 0.1.0  

published 10 hours ago by mikol <mikol@thinbox.org>
```

### supporting information:

 - `npm -v` prints: 6.1.0
 - `node -v` prints: v9.8.0
 - `npm config get registry` prints: https://registry.npmjs.org/
 - Windows, OS X/macOS, or Linux?: macOS 10.12.6
 - Network issues:
   - Geographic location where npm was run:
   - [ ] I use a proxy to connect to the npm registry.
   - [ ] I use a proxy to connect to the web.
   - [ ] I use a proxy when downloading Git repos.
   - [ ] I access the npm registry via a VPN
   - [ ] I don't use a proxy, but have limited or unreliable internet access.
 - Container:
   - [ ] I develop using Vagrant on Windows.
   - [ ] I develop using Vagrant on OS X or Linux.
   - [ ] I develop / deploy using Docker.
   - [ ] I deploy to a PaaS (Triton, Heroku).

**Comment by VictorQueiroz** [comment]
![screenshot from 2018-05-31 13-51-50](https://user-images.githubusercontent.com/5693176/40795912-d1447420-64d9-11e8-9d21-9db142913bd1.png)

The only work around I can get now is to create a private npm repository using a docker image like (https://hub.docker.com/r/verdaccio/verdaccio/) and consume packages from there until npm repository fix this problem. Verdaccio will use npm repository if it didn't find the package so you can safely publish your versions there and use the endpoint in your applications.

**Comment by mikol** [comment]
Appears to be working now.

**Comment by nickanna42** [comment]
Still happening for bellcurvejs package. I published v1.0.5. Still only able to install v1.0.4
